packages/web/src/common/components/PageContent.tsx

```diff
@@ -11,9 +11,10 @@ const PageContent = ({ children }: { children: React.ReactNode }) => {
   const path = usePathname();
 
   const isDevelopment = process.env.NEXT_PUBLIC_APP_MODE === "dev";
-  const productionReady = productionReadyPaths
-    .map(p => p.startsWith(path))
-    .some(p => p);
+  const productionReady =
+    (path in productionReadyPaths.exact ||
+      productionReadyPaths.startsWith.some(p => path.startsWith(p))) &&
+    !productionReadyPaths.exclude.includes(path);
```

Comment:
```suggestion
  const productionReady =
    (productionReadyPaths.exact.some(p => p.startsWith(path)) ||
      productionReadyPaths.startsWith.some(p => path.startsWith(p))) &&
    !productionReadyPaths.exclude.includes(path);
```

이렇게 해야 exact인 페이지들이 보여용

Reply:
구러냐 구러쿤 조아여